exporter/awskinesisexporter/internal/batch/aggregator.go

```diff
@@ -0,0 +1,75 @@
+package batch
+
+import (
+	"crypto/md5"
+
+	k "github.com/aws/aws-sdk-go/service/kinesis"
```

Comment:
I am rather against single value package name since it can be showed rather easily with should be locally scoped variables. 

I am fine if you'd like to rename it to `kpl ...` or leave it as `kinesis`